jsk_arc2017_common/config/box_sizes.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
> What is env settings?

env setting includes:
- object data
- shelf data
- cardboard box data
- tote data

Image file can be a config, too.